cmake/modules/FindLibDRM.cmake

```diff
@@ -30,9 +31,24 @@ find_package_handle_standard_args(LibDRM
                                   REQUIRED_VARS LIBDRM_LIBRARY LIBDRM_INCLUDE_DIR
                                   VERSION_VAR LIBDRM_VERSION)
 
+include(CheckCSourceCompiles)
+set(CMAKE_REQUIRED_INCLUDES ${LIBDRM_INCLUDE_DIR})
+check_c_source_compiles("#include <drm_mode.h>
+
+                         int main()
+                         {
+                           struct hdr_output_metadata test;
+                           test.metadata_type = 1;
+                         }
+                         " LIBDRM_HAS_HDR_OUTPUT_METADATA)
+
 if(LIBDRM_FOUND)
   set(LIBDRM_LIBRARIES ${LIBDRM_LIBRARY})
   set(LIBDRM_INCLUDE_DIRS ${LIBDRM_INCLUDE_DIR})
+  set(LIBDRM_DEFINITIONS -DHAVE_LIBDRM=1)
```

Comment:
Where is this used?

Reply:
Thanks for catching this, it is not used so I will drop it, we only need to know if `stuct hdr_output_metadata` is declared in libdrm headers (it will be in a future version after next sync from kernel headers).

Reply:
`HAVE_LIBDRM` have now been dropped